indexer/services/roundtable/src/tasks/update-wallet-total-volume.ts

```diff
@@ -1,50 +1,48 @@
 import { logger, stats } from '@dydxprotocol-indexer/base';
-import { PersistentCacheTable, WalletTable } from '@dydxprotocol-indexer/postgres';
+import {
+  PersistentCacheTable, WalletTable, PersistentCacheKeys, PersistentCacheFromDatabase,
+} from '@dydxprotocol-indexer/postgres';
 import { DateTime } from 'luxon';
 
 import config from '../config';
 
 const defaultLastUpdateTime: string = '2020-01-01T00:00:00Z';
-const persistentCacheKey: string = 'totalVolumeUpdateTime';
 
 /**
- * Update the total volume for each address in the wallet table.
+ * Update the total volume for each addresses in the wallet table who filled recently.
  */
 export default async function runTask(): Promise<void> {
   try {
     const start = Date.now();
-    const persistentCacheEntry = await PersistentCacheTable.findById(persistentCacheKey);
+    const persistentCacheEntry: PersistentCacheFromDatabase | undefined = await PersistentCacheTable
+      .findById(PersistentCacheKeys.TOTAL_VOLUME_UPDATE_TIME);
 
     if (!persistentCacheEntry) {
       logger.info({
-        at: 'update-address-total-volume#runTask',
-        message: `No previous totalVolumeUpdateTime found in persistent cache table. Will use default value: ${defaultLastUpdateTime}`,
+        at: 'update-wallet-total-volume#runTask',
+        message: `No previous ${PersistentCacheKeys.TOTAL_VOLUME_UPDATE_TIME} found in persistent cache table. Will use default value: ${defaultLastUpdateTime}`,
       });
     }
 
-    const lastUpdateTime = DateTime.fromISO(persistentCacheEntry
+    const lastUpdateTime: DateTime = DateTime.fromISO(persistentCacheEntry
       ? persistentCacheEntry.value
       : defaultLastUpdateTime);
-    let currentTime = DateTime.utc();
+    let windowEndTime = DateTime.utc();
 
     // During backfilling, we process one day at a time to reduce roundtable runtime.
-    if (currentTime > lastUpdateTime.plus({ days: 1 })) {
-      currentTime = lastUpdateTime.plus({ days: 1 });
+    if (windowEndTime > lastUpdateTime.plus({ days: 1 })) {
+      windowEndTime = lastUpdateTime.plus({ days: 1 });
     }
 
-    await WalletTable.updateTotalVolume(lastUpdateTime.toISO(), currentTime.toISO());
-    await PersistentCacheTable.upsert({
-      key: persistentCacheKey,
-      value: currentTime.toISO(),
-    });
+    await WalletTable.updateTotalVolume(lastUpdateTime.toISO(), windowEndTime.toISO());
```

Comment:
same as above, I would never expect this change to exist in `add update affiliate info roundtable`, please move into  a separate PR

Reply:
same as above